docker-entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+set -e
+
+if [ "$OPENFGA_MIGRATE_ON_INIT" = "1" ] && [ -n "$OPENFGA_DATASTORE_URI" ] ; then
```

Comment:
Introducing adhoc environment variables that are not owned by the binary itself is non-ideal here.